packages/sdk/src/modules/loggerModule/index.ts

```diff
@@ -1,14 +1,28 @@
 import { LoggerFactory, LoggerType } from "@vue-storefront/logger";
 import type { Module } from "../../types";
 import type { LoggerModuleConfig } from "./types";
+import { injectMetadata } from "./injectMetadata";
+import { isInvalidConfig } from "./utils";
 
 export const loggerModule = (options?: LoggerModuleConfig) => {
   const logger =
     options?.handler ?? LoggerFactory.create(LoggerType.ConsolaGcp, options);
 
+  if (isInvalidConfig(options)) {
+    console.warn(
+      "Both handler and options are provided to logger's options. Using handler, options will be ignored."
+    );
+  }
+
+  const loggerWithMetadata = injectMetadata(logger, {
+    alokai: {
+      context: "storefront",
+    },
+  });
+
   return {
     connector: {
-      ...logger,
+      ...loggerWithMetadata,
```

Comment:
It will not work, let's talk about it on the daily.

Reply:
As discussed, it stays as it is.